static/docs/use-cases/how-to-store-files-in-dvc.md

```diff
@@ -0,0 +1,129 @@
+# How to store files in DVC
+
+DVC allows storing and versioning files with git, without checking the file
```

Comment:
It is better to introduce the concept of data files and "sale" the idea that data files should be treated a special way.

Something like "In some cases, it is important to store and version **data files** along with code files. It is especially important in machine learning project when ML model files and source data files as much valuable as modeling code. However, Git was not designed to handle data file and binary files in general. ..."

Reply:
I think that would be out of scope of this small article, and would be more appropriate for the "DVC Philosophy".